src/blockdata/transaction.rs

```diff
@@ -364,46 +372,37 @@ impl TransactionPrefix {
                     .zip(self.outputs.iter())
                     .zip(tx_additional_pubkeys.iter())
                     .filter_map(|((i, out), tx_pubkey)| {
```

Comment:
:upside_down_face: We could use izip! (doesn't make a difference :stuck_out_tongue:)

Reply:
I don't want to introduce a dependency here with code that can probably be upstreamed easier :)